src/OrchardCore.Modules/OrchardCore.Forms/Services/IReCaptchaClient.cs

```diff
@@ -0,0 +1,9 @@
+using System.Threading.Tasks;
+
+namespace OrchardCore.Forms.Services
+{
+    public interface IReCaptchaClient
```

Comment:
You don't have to create an interface for everything. A class would have been sufficient here. We don't need abstractions for everything. You can register the class directly, and resolve it directly too. Even more that I assume the implementation is directly linked to the html that is generated outside of this interface. So maybe just call it `NoCatpcha` class (the name on Google).

Reply:
I will rmeove it if you insist, but I did this so that the types depending on `IReCaptchaClient` can be unit tested (where the test would inject a mocked version of `IReCaptchaClient`.

It's true that it's specific to the NoCaptcha thing from Google, so it makes sense to rename it to just that (it's not a generic service for bot detection in general).

Let me know.

Reply:
Yes it can be unit tested. But will you? ;) We can do integration testing though, more easily.

Reply:
I probably won't ;) 
But, I always thought it's good practice to write code that is testable - even if you don't actually write tests, because it forces you to write better code. Regardless, I will remove the interface and also rename the class to `NoCaptcha`.